docs/developers/index.md

```diff
@@ -439,6 +440,13 @@ The Ethereum network is made up of many nodes who run compatible client software
 
 ### General Learning {#general-learning}
 
+**EatTheBlocks -** **_Learn to build Ethereum Dapps & Solidity smart contracts with video tutorials
+
+- [Youtube - 1 new video / day](https://www.youtube.com/c/eattheblocks)
+- [Website](https://eattheblocks.com)
+
+- [ethereum.stackexchange.com](https://ethereum.stackexchange.com/)
```

Comment:
This looks to be a duplicate of line 452

Reply:
fixed :)